overlord/snapstate/snapmgr.go

```diff
@@ -488,10 +480,17 @@ func (m *SnapManager) NextRefresh() time.Time {
 	return m.nextRefresh
 }
 
-// RefreshSchedule returns the current refresh schedule.
+// RefreshSchedule returns the current refresh schedule as a string
+// suitable to display to a user.
 // The caller should be holding the state lock.
 func (m *SnapManager) RefreshSchedule() string {
-	return m.currentRefreshSchedule
+	_, scheduleStr, err := m.refreshSchedule()
+	// refreshSchedule should almost never return an error, except
+	// when the state is corrupted or in similar emergencies.
+	if err != nil {
```

Comment:
I still don't think we tend to do this, I think we still want an error out and for api.go to deal